__tests__/endpoints.test.js

```diff
@@ -252,3 +252,101 @@ describe('POST /api/articles/:article_id/comments', () => {
         })
     })
 })
+
+describe('PATCH /api/articles/:article_id', () => {
+    describe('main functionality', () => {
+        test('returns code 200 with the updated article', () => {
+            const testVote = { inc_votes: 7 };
+            return request(app)
+            .patch('/api/articles/12')
+            .send(testVote)
+            .expect(200)
+            .then((res) => {
+                expect(res.body.article).toMatchObject({
+                    article_id: expect.any(Number),
```

Comment:
You know exactly what the article_id should be, so we should be more specific here.